requirements.txt

```diff
@@ -2,3 +2,4 @@ pandas
 google-auth
 google-auth-oauthlib
 google-cloud-bigquery
+tqdm
```

Comment:
requirements.txt is only used by developers / CI testing. You need to add this to `setup.py` for it to actually get listed as a dependency.

As @maxim-lian says, we could make this an optional dependency. If you go that route, use the ["extras" section of setup.py](http://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies) ([example](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/1040e8b00eb63fe1dda38094cb4fe9c6efe55f04/bigquery/setup.py#L88).